mohawk/tests.py

```diff
@@ -689,20 +716,6 @@ def test_bewit_with_ext(self):
         expected = '123456\\1356420707\\kscxwNR2tJpP1T1zDLNPbB5UiKIU9tOSJXTUdG7X9h8=\\xandyandz'
         eq_(b64decode(bewit).decode('ascii'), expected)
 
-    def test_bewit_with_ext_and_backslashes(self):
```

Comment:
Could you leave this test in and just change the assertion? It should raise an error, right?